--- ifd-egate-0.05/Makefile.pcsc	2003-04-02 22:53:32.000000000 +0400
+++ ifd-egate-0.05/Makefile	2012-11-07 17:09:52.018800236 +0400
@@ -1,7 +1,7 @@
 libifd_egate.so: egate.o ifdhandler.o
 	$(CC) -shared $(LDFLAGS) -o $@ egate.o ifdhandler.o -lusb
 
-CFLAGS=-fPIC $(PCSC_CFLAGS) $(USB_CFLAGS) -g
+CFLAGS=-fPIC $(PCSC_CFLAGS) $(USB_CFLAGS) -g -I/usr/include/PCSC
 LDFLAGS=$(PCSC_LDFLAGS) $(USB_LDFLAGS)
 
 clean::
